application/src/main/java/org/togetherjava/tjbot/commands/tophelper/TopHelpersMessageListener.java

```diff
@@ -24,7 +24,7 @@ public final class TopHelpersMessageListener extends MessageReceiverAdapter {
      * @see <a href="https://www.regular-expressions.info/unicode.html#category">Unicode
      *      Categories</a>
      */
-    private static final Pattern UNCOUNTED_CHARS = Pattern.compile("\\p{C}");
+    public static final Pattern UNCOUNTED_CHARS = Pattern.compile("\\p{C}");
```

Comment:
package-private is enough. the tests are in the same package.

Reply:
Right. I missed that because they are different source sets